packages/block-editor/src/store/actions.js

```diff
@@ -978,10 +1259,10 @@ export function exitFormattedText() {
 /**
  * Action that changes the position of the user caret.
  *
- * @param {string} clientId     The selected block client ID.
- * @param {string} attributeKey The selected block attribute key.
- * @param {number} startOffset  The start offset.
- * @param {number} endOffset    The end offset.
+ * @param {string|Object} clientId     The selected block client ID.
```

Comment:
should we update the description here, or better, use a different attribute name for `clientId` if we aren't actually passing a `clientId`? I'm guessing we're looking at passing an actual block instead of a `clientId`, but I would be surprised if I had to do this: `block = action.clientId;`

Reply:
Right, I just haven't stared fixing all the doc blocks yet.

Reply:
➕ to update the docs and possibly the param name here. Also can we rename the passed object's property from `start` to `selectionStart` to map the name in state?